IdentityCore/src/cache/mac/MSIDMacACLKeychainAccessor.m

```diff
@@ -81,7 +82,15 @@ - (nullable instancetype)initWithTrustedApplications:(nullable NSArray *)trusted
         // To protect the underlying keychain API, a single queue is used even if multiple instances of this class are allocated.
         static dispatch_once_t s_once;
         dispatch_once(&s_once, ^{
-            s_synchronizationQueue = dispatch_queue_create("com.microsoft.msidmackeychaintokencache", DISPATCH_QUEUE_CONCURRENT);
+            if (dispatchQueue)
+            {
+                s_synchronizationQueue = dispatchQueue;
```

Comment:
We init static variable with a local variable. Should we add a separate **static method** to set the GCD queue? Otherwise, we can call this init 2 times with different queues, but only 1st one will be used and 2nd one will be ignored.

Reply:
Updated the PR